magma_access_gateway_installer/agw_installation_errors.py

```diff
@@ -13,13 +21,18 @@ class UnsupportedOSError(AGWInstallationError):
     """Exception raised when installation has been started on an OS other than Ubuntu."""
 
     def __init__(self):
+        logger.error("Invalid OS! \n Magma AGW can only be installed on Ubuntu! Exiting...")
         super().__init__("Invalid OS! \n Magma AGW can only be installed on Ubuntu! Exiting...")
 
 
 class InvalidNumberOfInterfacesError(AGWInstallationError):
     """Exception raised if number of available network interfaces is different from expected."""
 
     def __init__(self):
+        logger.error(
```

Comment:
Instead of copy pasting the message, can you create a variable for the text and refer to it in both places?

Reply:
Same goes for most errors